src/CarbonAware.WebApi/src/Controllers/CarbonAwareController.cs

```diff
@@ -41,7 +41,10 @@ public async Task<IActionResult> GetBestEmissionsDataForLocationsByTime(string l
             { CarbonAwareConstants.Best, true }
         };
 
-        return await GetEmissionsDataAsync(props);
+        _logger.LogInformation("Calling plugin GetBestEmissionsDataAsync with payload {@props}", props);
+
+        var response = await _aggregator.GetBestEmissionsDataAsync(props);
+        return Ok(response);
```

Comment:
before this commit, it looks like the code did something like so: `return response.Any() ? Ok(response) : NoContent();`

Reply:
Not just that. The integration tests are failing because they expected an `Ilist`, but they're now getting a single instance.